lib/UnixConfigLoader.php

```diff
@@ -12,9 +12,13 @@ class UnixConfigLoader implements ConfigLoader
     const MAXNS = 3;
     const MAXDNSRCH = 6;
     const RES_MAXNDOTS = 15;
+    const RES_MAXRETRANS = 30 * 1000;
+    const RES_MAXRETRY = 5;
+    const RES_TIMEOUT = 5 * 1000;
+    const RES_DFLRETRY = 2;
     const DEFAULT_OPTIONS = [
-        "timeout" => 3000,
-        "attempts" => 2,
+        "timeout" => self::RES_TIMEOUT,
+        "attempts" => self::RES_DFLRETRY,
```

Comment:
Could we please use readable names here?

Reply:
@kelunik they respond to ones used by resolv implementation, sure we can but wouldn't we want them to reflect original ones? BTW they should be private anyway.

Reply:
As long as it's not the config format, we don't have to take over any code style from there. `self::DEFAULT_ATTEMPTS` is much clearer.